tests/verkle/eip4762_verkle_gas_witness/test_withdrawals.py

```diff
@@ -46,10 +46,10 @@ def test_withdrawals(blockchain_test: BlockchainTestFiller, fork: str):
     }
 
     witness_check = WitnessCheck()
-    for address in [TestAddress, TestAddress2, env.fee_recipient]:
```

Comment:
There're no txs thus the coinbase doesn't receive any ETH. Hence, it won't appear in the witness.